flapjack/ext.py

```diff
@@ -1,4 +1,4 @@
-# Copyright 2017 Endless Mobile, Inc.
+# Copyright 2017, 2018 Endless Mobile, Inc.
```

Comment:
Are you going to update the copyright as files are changed, or all at once?

Reply:
I usually do it as files are changed, but I actually don't know what the best practice is.